xbmc/cores/VideoPlayer/VideoRenderers/FrameBufferObject.cpp

```diff
@@ -53,10 +53,14 @@ CFrameBufferObject::CFrameBufferObject()
 
 bool CFrameBufferObject::IsSupported()
 {
+#if HAS_GLES >= 2
```

Comment:
Mmm... Returning true to a non-existing extension by if'ing an hardcoded value?
You *really* hate IFDEF's ;)